code/__DEFINES/jobs.dm

```diff
@@ -71,6 +71,11 @@
 #define SQUAD_VATGROWN "Squad VatGrown"
 #define SILICON_AI "AI"
 
+/// ERT
+#define VETERAN "Veteran"
+#define VETERAN_CAPTAIN "Veteran Captain"
+#define SPECOPS "Special Operations Soldier"
```

Comment:
these are supposed to be used both for job name and the skill name iirc

Reply:
Just removed the defines , no ERT uses them.